src/controllers/cart_product.js

```diff
@@ -0,0 +1,147 @@
+const {products, cartProducts, users} = require("../sequelized/models")
+
+const addToCart = async (req, res) => {
+  const {productId, userId} = req.params;
+  const tokenId = req.userId
+
+  if (userId != tokenId){ //matching values only
+    return res.status(403).json({"message": "Access denied."})
+  }
+
+  if (!(productId.trim().length > 0 && userId.trim().length > 0)){
+    return res.status(400).json({"message": "Empty params are not allowed."})
```

Comment:
Message should be meaningfull something like `Required parameters not found`.

Reply:
There's no need for this check.
Below we are checking a `productId` and `userId` is number or not